README.adoc

```diff
@@ -22,11 +22,11 @@ Manage your SDLC in Spinnaker using the GUI (graphical user interface), or confi
 - Functions
 
 
-Facilitate GitOps, and grant developers control of provisioning infrastructure for apps. Use Spinnaker’s Clouddriver to deploy to all of the major public cloud providers and Kubernetes. You may even orchestrate configuration and firmware changes as part of Spinnaker delivery https://www.spinnaker.io/concepts/#firewall[pipelines].
+Facilitate GitOps, and grant developers control of provisioning infrastructure for apps. Use Spinnaker’s Clouddriver to deploy to all of the major public cloud providers and Kubernetes. You may even orchestrate configuration and firmware changes as part of Spinnaker delivery https://www.spinnaker.io/concepts/#pipeline[pipelines].
 
 = Why Do I Need Spinnaker?
 
-With Spinnaker, create a “paved road” for application delivery, with guardrails that ensure only valid infrastructure and configuration reach production.
+With Spinnaker, create a *paved road* for application delivery, with guardrails that ensure only valid infrastructure and configuration reach production.
```

Comment:
Nit, but the comma after "application delivery" is unneeded.  I know you didn't change that, but we can polish the grammar while the hood is open :)

Reply:
Good idea. Done.